packages/kit/types/hooks.d.ts

```diff
@@ -20,11 +20,21 @@ export interface GetSession<Locals = Record<string, any>, Session = any> {
 	(request: ServerRequest<Locals>): MaybePromise<Session>;
 }
 
+export interface HandleInput<Locals = Record<string, any>> {
+	request: ServerRequest<Locals>;
+	resolve: (request: ServerRequest<Locals>) => MaybePromise<ServerResponse>;
+}
+
 export interface Handle<Locals = Record<string, any>> {
-	(input: {
-		request: ServerRequest<Locals>;
-		resolve(request: ServerRequest<Locals>): MaybePromise<ServerResponse>;
-	}): MaybePromise<ServerResponse>;
+	(input: HandleInput<Locals>): MaybePromise<ServerResponse>;
+}
+
+export interface SequenceHandleInput<Locals = Record<string, any>> extends HandleInput<Locals> {
+	next: () => MaybePromise<ServerResponse>;
+}
+
+export interface SequenceHandle<Locals = Record<string, any>> {
+	(input: SequenceHandleInput<Locals>): MaybePromise<ServerResponse>;
```

Comment:
```suggestion
```

Reply:
Deleted the commit which added the `next` function.